tests/Avalonia.Markup.Xaml.UnitTests/Data/BindingTests.cs

```diff
@@ -34,10 +24,16 @@ public void Binding_With_Null_Path_Works()
                 var window = (Window)loader.Load(xaml);
                 var textBlock = window.FindControl<TextBlock>("textBlock");
 
-                window.DataContext = "foo";
                 window.ApplyTemplate();
 
+                window.DataContext = "foo";
                 Assert.Equal("foo", textBlock.Text);
```

Comment:
I don't think this is really the right place for this test - this test is testing that a `{Binding}` without a path works, which has nothing to do with `BindingOperations.DoNothing`. Also setting `DataContext` to `BindingOperations.DoNothing` seems a rather strange test...

Reply:
I totally understand that, and I agree it seems strange. When I wrote this, the idea was to test some more scenarios with bindings with null path, although thinking better it really doesn't belong here.

The main purpose for this is to define/illustrate the behavior of `DoNothing` with this PR, which in some cases may not seem to make sense, but they're cases which have to be handled (i.e. setting `DataContext` or a property in the `DataContext` to `DoNothing`).

I'll move this test elsewhere, but first I'd like to ask if you believe this behavior looks correct? `DoNothing` is a really special value, and I think setting a property in the view model to this value wouldn't make sense, but if someone does it, they will get the expected behavior I believe.

Reply:
Yeah I think setting a property on a view model to `DoNothing` would be a decent thing to unit test.